includes/Core/Assets/Assets.php

```diff
@@ -781,6 +789,39 @@ private function get_inline_dashboard_sharing_data() {
 			}
 		}
 
+		// Expose recoverable modules to the client.
+		$modules           = new Modules( $this->context );
+		$shareable_modules = $modules->get_shareable_modules();
+
+		foreach ( $shareable_modules as $module ) {
+			$owner_id = $module->get_owner_id();
```

Comment:
Not all modules have this method. We need to make sure that `$module` is an instance of the `Module_With_Owner` class, before invoking the `get_owner_id` method.